frontend/src/App.js

```diff
@@ -2,17 +2,20 @@ import React from 'react';
 import { Route, Routes } from 'react-router-dom';
 import Home from './pages/Home';
 import Profile from './pages/Profile';
+import Navbar from "./Navbar"
 
-function App() {
+function App(){
   return (
+    <>
+    <Navbar/>
     <div className='App'>
-      <p>HI</p>
       <Routes>
-        <Route path='' element={<Home />}/>
+        <Route path='/home' element={<Home />}/>
         <Route path='/profile' element={<Profile />}/>
       </Routes>
     </div>
+  </>
```

Comment:
Remove empty fragment lines and use <div> instead, nitpick but better to keep it same throughout the app

Also remove path '/home' for now and keep it as path=''. It's easier to just remove it for now because that way the app will render immediately showing home. We might change it later if we have time and want to show a login page first, but it's not hard to do when that time comes